pwa/src/components/deleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import "./deletemodal.css";
+import { Modal } from "@conductionnl/nl-design-system";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+interface DeleteModalProps {
+  handleDelete: any;
```

Comment:
This `prop` shouldn't be `:any`, I think it's always an function (and probably always a `void` function). So that would become something like:

`handleDelte: () => void`